src/game/game_init.h

```diff
@@ -30,7 +30,6 @@ enum ZBmodes {
     KEEP_ZBUFFER = 0,
     CLEAR_ZBUFFER = 1,
 };
-
```

Comment:
>:(

Reply:
Sorry @Arceveti broke into my house and did that